src/bot.ts

```diff
@@ -41,8 +41,19 @@ const onIssueComment = async (
     return `You are not allowed to request a tip. Only members of ${allowedGitHubOrg}/${allowedGitHubTeam} are allowed.`;
   }
 
-  const contributorAccount = parseContributorAccount(pullRequestBody);
-  const tipSize = getTipSize(tipSizeInput);
+  let contributorAccount: ContributorAccount;
+  try {
+    contributorAccount = parseContributorAccount(pullRequestBody);
+  } catch (error) {
+    return error.message;
+  }
+
+  let tipSize: string;
+  try {
+    tipSize = getTipSize(tipSizeInput);
+  } catch (error) {
+    return error.message;
```

Comment:
That covers only these 2 methods, but it doesn't for example `tipUser` or anything else.
I would propose to look into how `respondOnResult` writes the message in ALL cases, and make sure the trace isn't printed there